code/lib/instrumenter/package.json

```diff
@@ -45,7 +45,7 @@
   },
   "dependencies": {
     "@storybook/global": "^5.0.0",
-    "@vitest/utils": "^1.3.1",
+    "@vitest/utils": "^2.0.5",
```

Comment:
What's the breaking change in this package? Can it affect the instrumenter?

Reply:
No breaking changes